src/mudlet.cpp

```diff
@@ -2698,11 +2698,20 @@ void mudlet::doAutoLogin(const QString& profile_name)
             pHost->mSslTsl = (*it).tlsEnabled;
         }
     } else {
-        QFile file(qsl("%1/%2").arg(folder, entries.at(0)));
+        QFile file(qsl("%1%2").arg(folder, entries.at(0)));
         file.open(QFile::ReadOnly | QFile::Text);
         XMLimport importer(pHost);
         qDebug() << "[LOADING PROFILE]:" << file.fileName();
-        importer.importPackage(&file); // TODO: Missing false return value handler
+        importer.importPackage(&file);
+        if (auto [success, message] = importer.importPackage(&file); !success) {
+            pHost->postMessage(tr("[ ERROR ] - Something went wrong loading your Mudlet profile and it could not be loaded.\n"
+                "Try loading an older version in 'Connect - Options - Profile history' or double-check that %1 looks correct.").arg(file.fileName()));
+        
+            qDebug() << "mudlet::doAutoLogin: ERROR loading" << file.fileName() << "due to:" << message;
```

Comment:
```suggestion
            qDebug() << "mudlet::doAutoLogin: ERROR loading " << file.fileName() << " due to: " << message;
```

Reply:
As before, and I would have said:
```suggestion
            qDebug().nospace().noquote() << "mudlet::doAutoLogin(\"" << profileName << "\") ERROR - loading \"" << file.fileName() << "\" failed, reason: \"" << message < "\".";
```